inference-engine/tests/functional/inference_engine/ngraph_reader/negative_tests.cpp

```diff
@@ -158,9 +158,9 @@ TEST_F(NGraphReaderTests, ConvertNGraphFromIterator) {
     auto nGraph = reader.ReadNetwork(model, weights);
     CNNNetwork network(nGraph);
 
-    size_t layersCount = 0;
-    IE_SUPPRESS_DEPRECATED_START
-    for (auto it = network.begin(); it != network.end(); it++, layersCount++) {}
-    IE_SUPPRESS_DEPRECATED_END
-    ASSERT_EQ(layersCount, 2);
+    // size_t layersCount = 0;
```

Comment:
Looks like we should modify or remove this test.

Reply:
remove the commented out code.